BUILD.md

```diff
@@ -5,159 +5,159 @@ Main project target is to release the artifacts to one of central dependencies r
 (Maven central so far, but might be Bintray in the future). Unfortunately there is no so far any consistent way to 
 publish the project using TravisCI, so manual publishing is required.
```

Comment:
@andrii-kovalenko-ct but it seems that through [here](https://github.com/commercetools/commercetools-payment-integration-java/pull/41/files#diff-354f30a63fb0907d4ad57269548329e3R24) and [here](https://github.com/commercetools/commercetools-payment-integration-java/pull/41/files#diff-d1e37d2977ac59a5f763f1d9fa4ad259R15)  you're publishing to bintray automatically through travis? so what do you mean by this sentence?

> Unfortunately there is no so far any consistent way to publish the project using TravisCI, so manual publishing is required.